spec/controllers/reservations_controller_spec.rb

```diff
@@ -1376,80 +1320,61 @@
   end
 
   describe '#renew (PUT /reservations/:id/renew)' do
-    # Access: Admins, checkout persons, users if it's their own reservation
-    # Functionality:
-    # - sets @reservation
-    # - add @reservation.max_renewal_length_available.days to due_date
-    # - add 1 to @reservation.times_renewed
-    # - redirects to @reservation if you can't save
-    # - redirects to root_path if you can save
-    #     (is saving determined by equipment_model.max_renewal_times /
-    #      max_renewal_length?)
-
-    # TODO: Test circumstances under which renewal doesn't/shouldn't work
-
+    # SMELL: this mostly tests permissions
```

Comment:
I used `SMELL` to tag mostly [proper code smells](http://www.industriallogic.com/wp-content/uploads/2005/09/smellstorefactorings.pdf), but in some cases I wasn't able to determine the exact smell(s) so there are comments like this.

In this case, we are getting rid of controller tests that only test permissions in favor of testing `Ability` directly (see #1606), so we should either decide to get rid of this now or keep the comment so it can be gotten rid of in #1606.